@here/harp-olp-utils/lib/OlpDataProvider.ts

```diff
@@ -90,9 +94,14 @@ export class OlpDataProvider implements DataProvider {
         if (this.m_versionLayerClient === undefined) {
             throw new Error("OlpDataProvider is not connected.");
         }
-        return this.m_versionLayerClient.getTile(tileKey).then(response => {
-            // 204 - NO CONTENT, no data exists at the given tile. Do nothing.
-            return response.status === 204 ? Promise.resolve({}) : response.arrayBuffer();
-        });
+        return this.m_versionLayerClient
```

Comment:
Causing our terrain landmark example to print errors to the browser console.

https://render-verity.jenkins.release.in.here.com/view/VerityPreSubmit/job/verity-sdk-pre-submit/10274/Try_Me/
From top menu:
verity-examples/index 
Then from left menu:
Terrain -> landmark

VersionedLayerClient.js:85 Uncaught (in promise) TypeError: Cannot read property 'dataHandle' of undefined
    at VersionedLayerClient.<anonymous> (VersionedLayerClient.js:85)
    at Generator.next (<anonymous>)
    at fulfilled (VersionedLayerClient.js:22)

ConsoleChannel.ts:16 LandmarkTileDataSource: Cannot decode landmark tile: 332772335

Reply:
Thanks Craig for spotting, I've added an additional `catch` and also informed the OLP team of that regression.